### PR TITLE
WD-5607 - Update the projects directory

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done

Update the projects directory with the latest changes in the document.

## QA

- Open [the demo](https://canonical-com-1146.demos.haus/projects/directory)
- Check the copy matches the copy from [the project sheet](https://docs.google.com/spreadsheets/d/14Smfpq9ttrPt2jPx70NB3QTixH_6h7Ktnp1CihjT1So/edit#gid=0)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5607

### CODE REVIEW

File: templates/projects/directory.html [hunk 1]
```diff
@@ -35,292 +35,278 @@ <h1>Open source projects directory</h1>
...
-              <th scope="col">Contact email</th>
             </tr>
           </thead>
           <tbody>
             <tr>
-              <th scope="row">Candid authentication service</th>
-              <td>candid</td>
-              <td>Francesco Banconi</td>
-              <td>francesco.banconi<span class="sign sign--first">{ AT }</span>canonical<span class="sign sign--second">{ DOT }</span>com</td>
+            <th scope='row'>Candid authentication service</th>
```
Comment (akbarkz) [inline templates/projects/directory.html:42]: I think your formatter should be reconfigured. We use double quotes in HTML.

Comment (akbarkz) [reply templates/projects/directory.html:42]: Also, the indentation looks weird, `th` needs to be indented from `tr`, basically how it was before

Comment (anthonydillon) [reply templates/projects/directory.html:42]: Oh this mark up is generated by the spreadsheet but I can format it.

**Comment by anthonydillon** [comment]
Done, I know Prettier has support for jinja-html now. If you have a good set up I would be interested in seeing it? Might help the team?

**Review by akbarkz — APPROVED** [review]

**Pull request merged**